TestBenches/FileReadUtility.h

```diff
@@ -165,7 +165,7 @@ unsigned int compareMemWithMem(const MemType& memory_ref, const MemType& memory,
 
   unsigned int err_count = 0;
 
-  constexpr int width = (LSB >= 0 && MSB >= LSB) ? (MSB - LSB + 1) : MemType::getWidth();
+  constexpr int width = (LSB >= 0 && MSB >= LSB) ? (MSB + 1) : MemType::getWidth();
```

Comment:
@aryd This is another update from #333 that seems to have been undone.

Reply:
Restored this update.

Reply:
Restored these updates.